src/CircuitZoo/CircuitZoo.jl

```diff
@@ -49,12 +49,196 @@ function (circuit::Purify2to1)(purifiedL,purifiedR,sacrificedL,sacrificedR)
     apply!((sacrificedR,purifiedR),gate)
     measa = project_traceout!(sacrificedL, basis)
     measb = project_traceout!(sacrificedR, basis)
-    success = measa ⊻ measb == parity
+    success = (measa ⊻ measb == parity)
```

Comment:
Avoid modifying unrelated code. This is probably a leftover from some testing and refactoring.